vsphere-mixin/panels.libsonnet

```diff
@@ -0,0 +1,1472 @@
+        + g.panel.timeSeries.standardOptions.withUnit('percent'),
+
+      topDiskAvgLatencyHosts:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top avg disk latency by ESXi hosts',
+          targets=[t.topDiskAvgLatencyHosts],
+          description='The ESXi hosts with the highest average disk latency in the datacenter.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topPacketErrorRateHosts:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top packet errors by ESXi hosts',
+          targets=[t.topPacketErrorRateHosts],
+          description='The ESXi hosts with the highest percentage of packet errors in the datacenter.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('percent'),
+
+      hostCPUUsage:
+        commonlib.panels.generic.timeSeries.base.new(
+          'CPU usage',
+          targets=[t.hostCPUUsage],
+          description='The amount of CPU used by the ESXi host.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('rotmhz')
+        + g.panel.timeSeries.options.legend.withDisplayMode('table'),
+
+      hostCPUUtilization:
+        commonlib.panels.memory.timeSeries.usagePercent.new(
+          'CPU utilization',
+          targets=[t.hostCPUUtilization],
+          description='The CPU utilization percentage of the ESXi host.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('percent')
+        + g.panel.timeSeries.options.legend.withDisplayMode('table'),
+
+      hostMemoryUsage:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Memory usage',
+          targets=[t.hostMemoryUsage],
+          description='The amount of memory used by the ESXi host.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('mbytes')
+        + g.panel.timeSeries.options.legend.withDisplayMode('table'),
+
+      hostMemoryUtilization:
+        commonlib.panels.memory.timeSeries.usagePercent.new(
+          'Memory utilization',
+          targets=[t.hostMemoryUtilization],
+          description='The memory utilization percentage of the ESXi host.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('percent')
+        + g.panel.timeSeries.options.legend.withDisplayMode('table'),
+
+      hostModifiedMemory:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Modified memory',
+          targets=[t.hostModifiedMemoryBallooned, t.hostModifiedMemorySwapped],
+          description='The amount of memory that has been swapped or ballooned on the ESXi host.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('mbytes')
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.options.legend.withPlacement('right'),
```

Comment:
Personal take for this panel, I'd only want to know if the values are non-zero. Currently as-is it'd include ballooned and swapped memory stats for each hosts, even if 0

Reply:
@Dasomeone same as what I mentioned for dropped packets (although for this graph we would probably be safer in the lines staying consistent. I feel pretty comfortable changing this here).

Reply:
I'll change this for the VM panel as well.

Reply:
Sounds good, thank you Stefan! As mentioned in the other comment, don't be afraid to swapping to a bar chart or keep it you think the interrupted lines would be too frequent.